deploy/directadmin.sh

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env sh
+
+# DirectAdmin 1.58.2 API
+# This script can be used to deploy certificates to DirectAdmin
+#
+# User must provide login data and URL (incl. port) to DirectAdmin.
+# You can create login key, by using the Login Keys function
+# ( https://da.example.com:8443/CMD_LOGIN_KEYS ), which only has access to
+# - CMD_API_SSL
+#
+# Report bugs to https://github.com/Eddict/acme.sh/issues
+#
+# Values to export:
+# export DA_Api="https://remoteUser:remotePassword@da.example.com:8443"
+# export DA_Api_Insecure=1
+#
+# Set DA_Api_Insecure to 1 for insecure and 0 for secure -> difference is
+# whether ssl cert is checked for validity (0) or whether it is just accepted (1)
+#
+# Thanks to https://github.com/TigerP, creator of dnsapi/dns_da.sh
+# That script helped a lot to create this one
+
+########  Public functions #####################
+directadmin_deploy() {
+  _cdomain="$1"
+  _ckey="$2"
+  _ccert="$3"
+  _cca="$4"
+  _cfullchain="$5"
+
+  _debug _cdomain "$_cdomain"
+  _debug _ckey "$_ckey"
+  _debug _ccert "$_ccert"
+  _debug _cca "$_cca"
+  _debug _cfullchain "$_cfullchain"
+
+  _DA_credentials && _DA_setSSL
+  return 0
+}
+
+####################  Private functions below ##################################
+# Usage: _DA_credentials
+# It will check if the needed settings are available
+_DA_credentials() {
+  DA_Api="${DA_Api:-$(_readaccountconf_mutable DA_Api)}"
```

Comment:
dont use _readaccountconf_mutable.
Use _getdeployconf and _savedeployconf
eg: https://github.com/acmesh-official/acme.sh/blob/master/deploy/docker.sh#L22

Reply:
thanks. adjusted and commited